controllers/dscinitialization/dscinitialization_controller.go

```diff
@@ -98,7 +96,15 @@ func (r *DSCInitializationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		return reconcile.Result{}, err
 	}
 
-	if r.isManagedService() {
+	// Get platform
+	platform, err := deploy.GetPlatform(r.Client)
+	if err != nil {
+		return reconcile.Result{}, err
+
+	}
+
+	// Apply Rhods specific configs
+	if platform == deploy.ManagedRhods || platform == deploy.SelfManagedRhods {
```

Comment:
question: we do not need to handle `deploy.OpenDataHub` case?

Reply:
I don't think so, the old if block (`if r.isManagedService()`) would only be entered if the `managed-odh` resource was present, which OpenDataHub does not deploy AFAIK

Reply:
probably i did not make myself clear :smile: 
it is not based on the change for this IF block, but whether we need to handle the logic of ODH in controller.

Reply:
From what I remember from Monday's meeting, yes, the operator should handle both RHODS and ODH; I'm not sure what this would mean in the code here, so let's wait for @VaishnaviHire to pitch in

Reply:
@zdtsw  We do not need to handle deploy.OpenDataHub here since those resources are only required downstream

Reply:
got it, thanks!